tests/lib/net_buf/buf_simple/src/main.c

```diff
@@ -240,6 +240,34 @@ ZTEST(net_buf_simple_test_suite, test_net_buf_simple_add_be64)
 			  sizeof(be64), "Invalid 64 bits byte order");
 }
 
+ZTEST(net_buf_simple_test_suite, test_net_buf_simple_add_str)
+{
+	int ret;
+	char *str;
+
+	ret = net_buf_simple_add_str_len(&buf, "01234567", 5);
+	zassert_equal(ret, 0, "Adding a string should have succeeded");
```

Comment:
Use `zassert_ok` to check return value.

```suggestion
	zassert_ok(ret, "Adding a string should have succeeded");
```

Reply:
Great help from ztest for this very frequent use-case. Now adjusted.